docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -0,0 +1,66 @@
+---
+id: deploy-process-application
+title: Deploy and run your process application
+description: You can deploy your process application to a Zeebe cluster, or run your process application to test and debug it, and observe how it performs in a live environment..
+---
+
+import DeployImg from './img/process-applications/define-stages-deploy.png'
+import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
+import DeployErrorImg from './img/process-applications/deploy-error.png'
+
+You can deploy your process application to a Zeebe cluster, and run your process application to test and debug it, and observe how it performs in a live environment.
```

Comment:
I believe we're changing them from Zeebe clusters to Camunda clusters- please confirm

Reply:
I've removed the specification as it was superfluous anyway, changed to read "to a cluster"